.chloggen/configopaque-implement-BinaryMarshaler.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Should this be considered a breaking change given your comment here: https://github.com/open-telemetry/opentelemetry-collector/pull/9295#discussion_r1453668575

Reply:
You're right it should not.